databases/week3/mealsharing_db.sql

```diff
@@ -31,4 +31,20 @@ CREATE TABLE `review` (
 `created_date` DATE NOT NULL,
 PRIMARY KEY (`id`),
 CONSTRAINT `fk_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
-);
\ No newline at end of file
+);
+INSERT INTO meal (id, title, description, location, `when`, max_reservations, price, created_date)
+VALUES (1, 'Crawfish soup', 'Traditional soup cooked on crawfish tails. Served with homemade dumplings with crayfish necks', 'Bregnegaardsvej 2A, 2920 Charlottenlund', '2022-08-01 10:00:00', 2, 250.50, '2022-08-01');
```

Comment:
The id is auto increment, so you don't need to write it manually, sql will do it